src/WebJobs.Script.WebHost/Models/ContainerFunctionExecutionActivity.cs

```diff
@@ -69,5 +69,11 @@ public override int GetHashCode()
                 return hashCode;
             }
         }
+
+        public bool IsFunctional()
```

Comment:
It's not clear what a "functional" activity is

Reply:
I couldn't think of a better name :)
It just means the instance is doing useful work.